### PR TITLE
Adding Prow jobs to the repo.

### PR DESCRIPTION
**Pull request by ybettan**
Those scripts are expected to be executed by Prow running in the
kubernetes organization.

Signed-off-by: Yoni Bettan <yonibettan@gmail.com>

### CODE REVIEW

**Comment by ybettan** [comment]
Here is the matching PR for configuring Prow - https://github.com/kubernetes/test-infra/pull/27012

File: ci/prow/lint.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (qbarrand) [inline ci/prow/lint.sh:1]: I would not add the `.sh` extension to those files. Apart from that, LGTM

Comment (ybettan) [reply ci/prow/lint.sh:1]: Why not?

Comment (qbarrand) [reply ci/prow/lint.sh:1]: The practice of no extension is for all executables, not just shell scripts. See the [Google style guide](https://google.github.io/styleguide/shellguide.html#file-extensions).

Comment (ybettan) [reply ci/prow/lint.sh:1]: Have commented on the Prow PR - https://github.com/kubernetes/test-infra/pull/27012

Comment (ybettan) [reply ci/prow/lint.sh]: Fixed.

**Review by qbarrand — APPROVED** [review]

**Pull request merged**